burst-gradle-plugin/src/test/kotlin/app/cash/burst/gradle/TestInterceptorGradlePluginTest.kt

```diff
@@ -22,23 +22,15 @@ import assertk.assertions.isEqualTo
 import assertk.assertions.isNotNull
 import assertk.assertions.isNull
 import org.gradle.testkit.runner.TaskOutcome
-import org.junit.BeforeClass
 import org.junit.Test
 
 class TestInterceptorGradlePluginTest {
-  companion object {
-    private val tester = GradleTester("interceptor")
-
-    @BeforeClass
-    @JvmStatic
-    fun beforeClass() {
-      val result = tester.cleanAndBuildAndFail(":lib:test")
-      assertThat(result.outcome).isEqualTo(TaskOutcome.FAILED)
-    }
-  }
-
   @Test
   fun happyPath() {
+    val tester = GradleTester("interceptor")
+    val result = tester.cleanAndBuildAndFail(":lib:test")
+    assertThat(result.outcome).isEqualTo(TaskOutcome.FAILED)
```

Comment:
These assertion seems redundant. Gradle's `buildAndFail` will already throw if the build did not fail.

Reply:
Nuked!